functest/deposit_migration_test.go

```diff
@@ -0,0 +1,136 @@
+package functest
+
+import (
+	"github.com/insolar/insolar/testutils"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+const insolarMigrationMember1 = "migration_daemon_0_member_keys.json"
+
+func TestMigrationTokenNotInTheList(t *testing.T) {
+	ba := testutils.RandomString()
+	_, err := signedRequest(&migrationAdmin,
+		"deposit.migration",
+		map[string]interface{}{"amount": "1000", "ethTxHash": "TxHash", "migrationAddress": ba})
+	require.Contains(t, err.Error(), "this migration daemon is not in the list")
+}
+
+func TestMigrationToken(t *testing.T) {
```

Comment:
You need the test that will check that something was created after your migration. And that all creates right. I'm talking about all deposits fields. 
And this test should be on the top.

Reply:
We will add the test implementation after the getBalance enters the master with a separate pullrequest. Without this, the test is not done

Reply:
Add task for that